lib/config/config-initializer.js

```diff
@@ -305,7 +306,7 @@ function promptUser(callback) {
             type: "list",
             name: "styleguide",
             message: "Which style guide do you want to follow?",
-            choices: [{name: "Google", value: "google"}, {name: "Airbnb", value: "airbnb"}, {name: "Standard", value: "standard"}],
+            choices: [{name: "Google", value: "google"}, {name: "Airbnb", value: "airbnb"}, {name: "Airbnb Base (no react)", value: "airbnb"}, {name: "Standard", value: "standard"}],
```

Comment:
I think you need to change the value of "AirbnbBase (no react)" to `airbnb-base`, otherwise this won't work.